model/openapi.yml

```diff
@@ -490,7 +490,42 @@ components:
             yaml: "timeout"
         signed:
           $ref: "#/components/schemas/actionSignature"
-
+    policyData:
```

Comment:
No go code has been generated for this because it is not linked as part of an action payload at the moment.

We can likely use `oneOf` to generate fleet-server's structs in order to be backwards compatible if that's required